xla/stream_executor/gpu/gpu_types.h

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 #ifndef XLA_STREAM_EXECUTOR_GPU_GPU_TYPES_H_
 #define XLA_STREAM_EXECUTOR_GPU_GPU_TYPES_H_
 
+#include <string>
 #if TENSORFLOW_USE_SYCL
```

Comment:
nit: style requires a space between std headers and everything else

Reply:
fixed